### DISCUSSION TITLE
Set Date Updated and User Updated on creation

### DISCUSSION THREAD

**Discussion by TheAzack9** [directus/directus#5835] [Archive]
As it works by default will only Date Created and User Created be set on creation and Date Updated and User Updated is null until the first update happens. What i would want is for the Date Updated and User Updated to be set to the same as Created when the element is created in the collection.

If you want to display in a preset when the element was last updated do you have to include Date Created in the case where the element hasn't been updated yet after creation, which feels cumbersome. Is it possible to change this behaviour locally somehow? (I'm not sure if it is a big enough issue that i want to make it a feature request, but imo would it make more sense that Date Updated / User Updated was the same as the Date Created / User Created as initial value instead of undefined.)

Sorry if this was horribly explained 😅  :) 

This just feels odd to me tho, but is really a minor nit pick :) 

![image](https://user-images.githubusercontent.com/2140448/119262379-4b455900-bbdb-11eb-97de-c4750184f119.png)

**Comment by benhaynes** [comment]
Agree! The admin can do this, but I think we should have better defaults. Basically, you can set the date/user updated fields to use both created/updated options under the field setup. I prefer to keep updated separate from create as an option so that you have all possibilities.

**Reply by TheAzack9** [reply]
Amazing! Thanks :)

**Reply by TheAzack9** [reply]
I can't find a way to set the User Updated, but it isn't the most important one for me anyways ;) Having the Date Updated being set solves the issue <3

**Reply by rijkvanzanten** [reply]
@TheAzack9 3d3a508880f0fd51df8a8f05b990c0687fca53f5 😘
